cli/operator/node.go

```diff
@@ -118,16 +114,32 @@ var StartNodeCmd = &cobra.Command{
 			logger.Fatal("could not setup db", zap.Error(err))
 		}
 
-		nodeStorage, operatorData := setupOperatorStorage(logger, db)
+		var operatorPrivKey keys.OperatorPrivateKey
+		if cfg.KeyStore.PrivateKeyFile != "" {
+			operatorPrivKey, err = keys.PrivateKeyFromFile(cfg.KeyStore.PrivateKeyFile, cfg.KeyStore.PasswordFile)
```

Comment:
I rather we take out the file logic from the keys package. read the files here and pass raw strings into the function.

Reply:
@y0sher I don't mind. Do you think we should also extract the JSON decryption logic with `keystorev4.New().Decrypt` or something like the below looks fine to you?

```go
if cfg.KeyStore.PrivateKeyFile != "" {
	// nolint: gosec
	encryptedJSON, err := os.ReadFile(cfg.KeyStore.PrivateKeyFile)
	if err != nil {
		logger.Fatal("could not read PEM file", zap.Error(err))
	}

	// nolint: gosec
	keyStorePassword, err := os.ReadFile(cfg.KeyStore.PasswordFile)
	if err != nil {
		logger.Fatal("could not read password file", zap.Error(err))
	}

	operatorPrivKey, err = keys.PrivateKeyFromEncryptedJSON(encryptedJSON, string(keyStorePassword))
	if err != nil {
		logger.Fatal("could not extract operator private key from file", zap.Error(err))
	}
} else {
	operatorPrivKey, err = keys.PrivateKeyFromString(cfg.OperatorPrivateKey)
	if err != nil {
		logger.Fatal("could not decode operator private key", zap.Error(err))
	}
}
```

Reply:
I fixed this here https://github.com/bloxapp/ssv/pull/1274/commits/23cb56903dd4b83c5e4071e188e9462e8c123b15